src/redux/books/bookStoreApi.js

```diff
@@ -0,0 +1,40 @@
+const API_ID = '60vWjx3DsVdMxCaJTgl8';
+const URL = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${API_ID}/books/`;
+
+export const getBooks = async () => {
+  const requestOptions = {
+    method: 'GET',
+    redirect: 'follow',
+  };
+  const result = await fetch(URL, requestOptions);
+  try {
+    const books = await result.json();
+    return books;
+  } catch (error) {
+    return {};
+  }
+};
+
+export const createBook = async (book) => {
+  const headers = new Headers();
+  headers.append('Content-Type', 'application/json');
+  const newBook = JSON.stringify(book);
+  const requestOptions = {
+    method: 'POST',
+    headers,
+    body: newBook,
+    redirect: 'follow',
+  };
+  const result = await fetch(URL, requestOptions);
+  return result.status === 201;
+};
+
+export const deleteBook = async (id) => {
+  const requestOptions = {
+    method: 'DELETE',
+    redirect: 'follow',
+  };
+  const response = await fetch(`${URL}${id}`, requestOptions);
+  const result = await response.text();
+  return result === 'The book was deleted successfully!';
+};
```

Comment:
- [ ]  `Remove` book features should work the same as before even after the refactoring to use the API has been done currently, `remove` is not working please fix that

Reply:
The button does work, it just needs to be pressed hard, bc of the reset style it has, it got difficult to press, the button is kinda crammed there, but it doesn't really matter because next step is styling and I would fix it there. The button does erase, I tested it many times before sending for review.

Reply:
Hello @sheylaPozo,

Thanks for reaching out  if your button works ask for another CR
and let the next reviewer know that because its requirements for
this project to be approved.

Thanks

Reply:
thank you for reaching out, yes I have been running some tests and it works, I will submit for a review, thank you.